plugins/ooni/main.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "OONI",
+    "name": "ooni",
```

Comment:
This was added in the `temporary hack` commit. Is this something that should be reverted before merging?